### ISSUE TITLE
NewPipe yt-more-params-innertube-requests Disappeared from my phone

### ISSUE THREAD

**Issue by boognish-rising** [TeamNewPipe/NewPipe#8360]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

So I was following the YouTube age bypass restriction issue very closely about a month or so ago and the ultimate solution was arrived at in the form of the above referenced debugging APK (NewPipe yt-more-params-innertube-requests) and I'd noticed that at least for a while that whatever fix had been included in that APK had not made it to the official release. So for a better or worse I began to more or less exclusively use that version of the app even though I know it wasn't probably the wisest decision and what this resulted in was that my history and playlists created during that time were only housed there. Fast forward to about 20 minutes ago when I noticed that it was no longer on my homepage/ desktop on my phone and after looking a little bit deeper have realized that it is nowhere to be found on my device anywhere. I know for a fact that I didn't delete it, so I figured I would come here and inquire as to whether or not it would even be possible for you guys to remotely delete this package from my device. I wouldn't think so but then again I have no idea what happened to it. So here I am posing this question to you, even though I know how ridiculous it sounds. I know that I shouldn't have been relying on it as it was not an official release and it was only debugging APK but it worked and you know what they say, "if it's not broken, don't fix it."  I'm perfectly fine with switching over to the official release. My only objection is that I could have exported my history, playlists, etc. and imported them into the official release. But now I don't have that option. I did look through the first couple pages of issues and didn't see anything remotely similar to my issue here, but I apologize if I didn't look far enough. 

PS I did review the contribution guidelines but seeing as how this is not a contribution nor is it a feature request or an issue that I'm raising, more like just a simple straightforward question, I don't _think_ that I need to provide anything further like package information, Android version, device model, etc. but if I'm incorrect in that assumption just let me know and I will be glad to provide anything further that would be necessary or helpful. Everything that I read in the contribution guidelines pertained to things outside of a simple question, they were specific to issues being raised and feature requests etc. But again, if I'm mistaken just let me know and I will correct the error of my ways. 

### Additional information

Here's a picture of Lil baby GusGus. She is called GusGus because we originally thought her to be a male and when I learned of the mistake, didn't have the heart to change her name so she mostly just goes by Lil Baby these days. She likes to haz treats and doesn't like when I have to go on trips, which you may have deduced from the fact that she planted herself in my suitcase whilst I was trying to pack it.
("pictures of cats" was specifically mentioned under Additional Information and I'm always happy to share pics of the Lil Baby)

![COVER_IMAGE2642940352872685821](https://user-images.githubusercontent.com/76668060/167059127-b9bacecb-b6ce-4c57-881e-f9b511a1eb2b.jpg)

**Comment by opusforlife2** [comment]
Ah... well... despite the cat picture...

This has nothing to do with Newpipe, sorry. Something else in your personal setup could have broken, whether it was your OS, or something else. Or it could have been an evil maid/dumb baby attack. Who knows?

**Comment by boognish-rising** [comment]
Thanks for confirming what I suspected, i.e., that you guys had nothing to do with it. It's still a bit of a mystery. Well I can't rule out evil maids or dumb baby attacks, I was able to find a backup of the APK and it turns out I did export my history, playlists, and settings etc. not too long prior to the package disappearing so I was able to retain most of it. Anyway, thanks again for confirming that there wasn't some grand conspiracy.

Viva la NewPipe!
